Src/Plugins/Tests.Security/Validators/SEC101_046.DiscordApiCredentialsValidatorTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    public class DiscordApiCredentialsValidatorTests
+    {
+        [Fact]
+        public void DiscordApiCredentialsValidator_Test()
+        {
+            string fingerprintText = "";
+            if (string.IsNullOrEmpty(fingerprintText))
+            {
+                return;
+            }
+
+            string message = null;
+            ResultLevelKind resultLevelKind = default;
+            var fingerprint = new Fingerprint(fingerprintText);
+            var keyValuePairs = new Dictionary<string, string>();
+
+            DiscordApiCredentialsValidator.IsValidDynamic(ref fingerprint,
+                                               ref message,
```

Comment:
>ref [](http://example.com/codeflow?start=47&length=3)

pls, align by ref fingerprint (check other files to see the pattern)

Reply:
fixed